examples/templates/aws-linux/main.tf

```diff
@@ -36,6 +36,22 @@ variable "region" {
   }
 }
 
+variable "instance_type" {
+  description = "What instance type should your workspace use?"
+  default     = "2C/1G: t3.micro"
```

Comment:
I've had a few instances where defining the `default` skipped the prompt. I'm guessing this is not the case here? (Just checking.)

Reply:
I don't recall it being the case when I tested it!